src/test/java/com/ite/itea/ecommerce/domain/retail/CustomProductTest.java

```diff
@@ -0,0 +1,46 @@
+package com.ite.itea.ecommerce.domain.retail;
+
+import com.ite.itea.ecommerce.domain.core.EuroPrice;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CustomProductTest {
+
+    @Test
+    void shouldCalculatePrice() {
```

Comment:
Suggestion: `priceShouldBeSumOfPartsPrices()`
That way we don't imply that a calculation happens when the method is called (it might be, e.g., lazily calculated once, or one of a few other options).

Reply:
My intention here was to say calculate/get the price but without describing how. if you are interested how it is calculated then you can have a closer look into the test itself . But I totally agree with your point.